packages/pages/src/generate/features/features.ts

```diff
@@ -0,0 +1,66 @@
+import path from "path";
+import { CommandModule } from "yargs";
+import { Path } from "../../common/src/project/path.js";
+import { ProjectFilepaths } from "../../common/src/project/structure.js";
+import { getTemplateFilepaths } from "../../common/src/template/internal/getTemplateFilepaths.js";
+import { loadTemplateModules } from "../../common/src/template/internal/loader.js";
+import { createFeaturesJson } from "./createfeaturesjson.js";
+
+type FeaturesArgs = Pick<
+  ProjectFilepaths,
+  "templatesRoot" | "sitesConfigRoot" | "scope"
+>;
+
+const handler = async ({
+  templatesRoot,
+  sitesConfigRoot,
+  scope,
+}: FeaturesArgs): Promise<void> => {
+  const templateRootAbsolutePath = path.join(process.cwd(), templatesRoot);
+  const templateFilepaths = getTemplateFilepaths(
+    scope
+      ? [
+          new Path(path.join(templateRootAbsolutePath, scope)),
+          new Path(templateRootAbsolutePath),
+        ]
+      : [new Path(templateRootAbsolutePath)]
+  );
+  const templateModules = await loadTemplateModules(
+    templateFilepaths,
+    true,
+    false
+  );
+  const featuresFilepath = path.join(
+    process.cwd(),
+    sitesConfigRoot,
+    scope ?? "",
+    "features.json"
+  );
+  await createFeaturesJson(templateModules, featuresFilepath);
+};
+
+export const featureCommandModule: CommandModule<unknown, FeaturesArgs> = {
+  command: "features",
+  describe: "Generates features.json file",
+  builder: (yargs) => {
+    return yargs
+      .option("templatesRoot", {
```

Comment:
Are the two flags that aren't scope necessary? It could be nice but we're adding functionality we probably don't need.

Reply:
I was thinking we do expose the config option `templatesRoot` and `sitesConfigRoot` in our vite-plugin, so it may be possible that people can configure their root file paths differently from the defaults, which would affect the filepaths in how we scope the build (in which case I will update `dev` command to include these two options too). If you think that's not super necessary still, I can remove it here

Reply:
Makes sense, but in reality we should unexpose those in the plugin as it won't actually work. It's because you can't import templates with a variable prefix since Vite statically replaces the import.meta call.

So I think we should remove these options for now here and in the plugin. Better yet, don't allow any options in the plugin.

Reply:
Okay, I will remove the options for features in this pr. We can probably address the plugin options in the next PR I put up for build command